transports/http/endpoints/api/access/endpoints.go

```diff
@@ -13,15 +13,15 @@ import (
 )
 
 type handler struct {
-	service users.UserService
-	log     *zerolog.Logger
+	services *domain.Services
```

Comment:
just for the future, instead of attaching all the services,
I would rather prefer to add replace service with dedicated fields like:
```suggestion
	userService users.UserService
	rateService rates.RatesService
```